docs/Developers/Testing_coala_change_on_Bears.rst

```diff
@@ -0,0 +1,103 @@
+Testing changes to coala on coala-bears
+=======================================
+
+When large changes are made to the coala core, they should be tested
+on the coala-bears to confirm that they work as expected and don't
+break anything on the bears' side.
+
+You can do this by changing the coala dependency for the bears to
+be the one consisting of the changes. In order to do this, you need to create
+a new branch on your coala-bears fork, update the coala requirement to point to
+your changes and run CI on it.
+
+Set up a new testing branch
+---------------------------
+
+Assuming you have already made changes to coala through a new branch, you can
+use that branch to install coala by editing requirements.txt in coala-bears.
+Let's set-up a new branch on coala-bears for these changes:
+
+If you have not already, clone the coala-bears repository by running:
+
+::
+
+    $ git clone -o upstream https://github.com/coala/coala-bears
+
+.. note::
+    ``-o upstream`` sets the remote name of the original coala-bears repository
+    as upstream.
+
+Now, navigate to the directory where coala-bears is located.
+
+::
+
+    $ cd coala-bears
+
+This tutorial assumes you are working on your own fork. To fork the repository,
```

Comment:
Change `To fork the repository` to `If you haven't done so` improves the readability.